R/wwinference.R

```diff
@@ -350,40 +370,41 @@ fit_model <- function(compiled_model,
 #' This function returns a list of MCMC settings to pass to the
 #' `cmdstanr::sample()` function to fit the model. The default settings are
 #' specified for production-level runs, consider adjusting to optimize
```

Comment:
I don't think the "consider adjusting for speed while iterating" is needed and it might be a bit confusing (since one of the configurable things is the number of iterations (in a different sense)

```suggestion
#' specified for production-level runs.
```